### PR TITLE
add Audience section and Audience Overview chart ANA-343

### PR DESCRIPTION
**Pull request by federicoweber**
### Purpose
To add the Audience section and Audience Overview chart.

#### Staging Deployment

_This repo is CI/CD enabled and staging deployment should be available at:
https://<branch_name>.analyze.buffer.com :smile:_

### CODE REVIEW

File: packages/profile-selector/middleware.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { actionTypes } from '@bufferapp/async-data-fetch';
 import { actions as performanceActions } from '@bufferapp/performance-tracking';
 import { actions as profilesActions, actionTypes as profileActionTypes } from './reducer';
 
-const INSIGHTS_PATH_REGEX = /(overview|posts|answers)\/?(.*)$/;
+const INSIGHTS_PATH_REGEX = /(overview|posts|answers|audience)\/?(.*)$/;
```
Comment (msanroman) [inline packages/profile-selector/middleware.js]: We should find another name for this, like `PAGES_FOR_SINGLE_PROFILES_REGEX` 😁

File: packages/web/components/InsightsPage/index.jsx [hunk 2]
```diff
@@ -12,6 +12,7 @@ import { Toolbar } from '@bufferapp/analyze-shared-components';
 import PostsTab from '../PostsTab';
 import OverviewTab from '../OverviewTab';
 import AnswersTab from '../AnswersTab';
+import AudienceTab from '../AudienceTab';
```
Comment (msanroman) [inline packages/web/components/InsightsPage/index.jsx]: Maybe this is no longer the `InsightsPage`, as Answers is no longer under the "Insights" menu.

**Comment by federicoweber** [comment]
I'm closing this and will manually cherry-pick it into `feature/audience` to keep things nice and clean on that branch.

**Pull request closed without merging**